src/metaObject.cxx

```diff
@@ -830,14 +830,13 @@ DistanceUnits(const char * _distanceUnits)
 const char * MetaObject::
 AnatomicalOrientationAcronym(void) const
   {
-  static char str[10];
   int i;
   for(i=0; i<m_NDims; i++)
     {
-    str[i] = MET_OrientationTypeName[m_AnatomicalOrientation[i]][0];
+    m_OrientationAcronym[i] = MET_OrientationTypeName[m_AnatomicalOrientation[i]][0];
```

Comment:
I suggest setting this variable when m_AnatomicalOrientation is set.  That way this can remain const without using a mutable variable.

s